src/libs/ajax.js

```diff
@@ -1388,7 +1388,7 @@ const Apps = signal => ({
             name: diskName,
             size: diskSize,
             labels: {
-              saturnApplication: 'galaxy',
+              saturnApplication: appType,
               saturnWorkspaceNamespace: namespace,
               saturnWorkspaceName: workspaceName
```

Comment:
worth checking with the IA team but I think `saturn` is a ye-olde name for "terra" and might be worth migrating away from?

Reply:
Yeah, these labels are used in the backend for filtering based on workspace. They are stored in a label map, and it would be a non-trivial to make this change while maintaining backwards compatibility (it would require updating the labels on all non-deleted runtimes)